scripts/llnl/common_build_functions.py

```diff
@@ -272,7 +272,7 @@ def build_and_test_host_config(test_root,host_config, report_to_stdout = False,
     bld_output_file =  pjoin(build_dir,"output.log.make.txt")
     print("[starting build]")
     print("[log file: %s]" % bld_output_file)
-    res = sexe("cd %s && make -j 16 VERBOSE=1 " % build_dir,
+    res = sexe("cd %s && make -j 8 VERBOSE=1 " % build_dir,
```

Comment:
what was the reason for this?

Reply:
This was what it took for the GCC image to stop imploding